src/filters/fb_network_builder.rs

```diff
@@ -156,33 +156,55 @@ impl<'a> FlatSerialize<'a, EngineFlatBuilder<'a>> for NetworkFilterListBuilder {
                     None
                 };
 
-                for tokens in multi_tokens {
-                    let mut best_token: ShortHash = 0;
-                    let mut min_count = total_number_of_tokens + 1;
-                    for token in tokens {
-                        let token = to_short_hash(token);
-                        match tokens_histogram.get(&token) {
-                            None => {
-                                min_count = 0;
-                                best_token = token
-                            }
-                            Some(&count) if count < min_count => {
-                                min_count = count;
-                                best_token = token
```

Comment:
does the speedup come from removing this `tokens_histogram` check for token groups of length 1, or from removing the extra `Vec` allocations by using `FilterTokens` instead? The former could be done without a breaking change; just trying to understand the options we have

Reply:
I've returned the original method to maintain API compatibility.